Source/Extensions/Blazorise.SignaturePad/wwwroot/signaturepad.js

```diff
@@ -136,6 +137,9 @@ export function undo(element, elementId) {
         if (data) {
             data.pop();
             instance.sigpad.fromData(data);
+
+            const dataURL = getImageDataURL(instance.sigpad, instance.options);
+            instance.dotNetAdapter.invokeMethodAsync("NotifyValue", dataURL);
```

Comment:
What if this returned the updated dataUrl instead of interoping again?
Although I'm not sure there's a big difference between making blazor wait for js interop response vs invoking a new one.

Reply:
That's a good idea.

Reply:
Done